app/controllers/authenticated/YourEmployerController.scala

```diff
@@ -65,11 +65,11 @@ class YourEmployerController @Inject()(
           taiService.employments(request.nino.get, taxYears.head).flatMap {
             employments =>
               if (employments.nonEmpty) {
-                val employerName = if (employments.forall(p => p.endDate.isDefined)) employments.head.name else employments.filter(p => p.endDate.isEmpty).head.name
+                val employerNames: Seq[String] = employments.map(_.name)
                 for {
```

Comment:
:+1: well that's a lot simpler :)

Reply:
I may be misunderstanding this but doesn't this include inactive employers?  Not sure I understand the data enough yet.

Reply:
Yes, this includes the inactive employments as well, so this will now show all employments for the year